signals.go

```diff
@@ -99,7 +99,7 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 				}
 			}
 		default:
-			logrus.Debugf("sending signal to process %s", s)
+			logrus.Debugf("forwarding %s to %d", unix.SignalName(s.(unix.Signal)), pid1)
```

Comment:
Looks like `unix.SignalName()` can potentially return an empty string if it doesn't know the signal; https://github.com/golang/sys/blob/15123e1e1f71fb044f0ebc6b23b37d9744890bf9/unix/syscall_unix.go#L67-L75

(It's of course only a debug log), but that could mean that (e.g.) one of the `SIGRTMIN` signals won't be printed. I guess we could add a wrapper and in that case just print the numeric value?

Reply:
You're quite right, `unix` does not know about signals > 31, and will return an empty string.

Since this is a debug print, I don't want to write anything more complicated than the one liner. So, I have modified it to print both signal number and the name, if known.

Looks like this:
```
    DEBU[0000]signals.go:103 main.(*signalHandler).forward() forwarding signal 23 (SIGURG) to 891345
    DEBU[0020]signals.go:103 main.(*signalHandler).forward() forwarding signal 45 () to 891345
    DEBU[0020]signals.go:103 main.(*signalHandler).forward() forwarding signal 23 (SIGURG) to 891345
    DEBU[0020]signals.go:103 main.(*signalHandler).forward() forwarding signal 23 (SIGURG) to 891345
```

Reply:
Yes, that looks like a good middle-ground 👍  (I agree to not over-complicate it, as it's just for debugging currently)